scripts/demography-explorer/metrics.csv

```diff
@@ -1,33 +1,33 @@
-Sex ratio at birth,Sex ratio,sex_ratio__none__${age_group__csv_slug}__${projection__slug},sex_ratio__none__full__${projection__slug},"​​The number of male births per 100 female births. Biological birth ratios are slightly male-biased, with an expected ratio of 105 male births per 100 female births. ${projection__subtitle}",LineChart,,,true,,,all,at_birth,records low medium high,Numeric,,,102;103;104;105;106;107;108;109,BrBG,101
-Sex ratio ${age_group__title_suffix},Sex ratio,sex_ratio__none__${age_group__csv_slug}__${projection__slug},sex_ratio__none__full__${projection__slug},"The number of male individuals per 100 female individuals, ${age_group__title_suffix}. ${projection__subtitle}",LineChart,,,true,,,all,5 10 15 20 30 40 50 60 70 80 90 100_and_over,records low medium high,Numeric,,,,Blues,
-Total ${sex__title_prefix} dependency ratio,Dependency ratio,dependency_ratio_total__${sex__slug}__none__${projection__slug},dependency_ratio_total__${sex__slug}__full__${projection__slug},The ratio of the young ${sex__title_prefix} population (under age 15) and elderly population (aged 65 and over) to the working-age ${sex__title_prefix} population (aged 15 to 64). ${projection__subtitle},LineChart,,0,true,,,all female male,dependency_total,records low medium high,Numeric,%,%,40;50;60;70;80;90;100;110,GnBu,
-${sex__title_prefix} youth dependency ratio,Dependency ratio,dependency_ratio_child__${sex__slug}__none__${projection__slug},dependency_ratio_child__${sex__slug}__full__${projection__slug},The ratio of the young ${sex__title_prefix} population (under age 15) to the working-age ${sex__title_prefix} population (aged 15 to 64). ${projection__subtitle},LineChart,,0,true,,,all female male,dependency_child,records low medium high,Numeric,%,%,25;30;40;50;75;100;110,GnBu,
-${sex__title_prefix} old-age dependency ratio,Dependency ratio,dependency_ratio_old__${sex__slug}__none__${projection__slug},dependency_ratio_old__${sex__slug}__full__${projection__slug},The ratio of the elderly ${sex__title_prefix} population (aged 65 and over) to the working-age ${sex__title_prefix} population (aged 15 to 64). ${projection__subtitle},LineChart,,0,true,,,all female male,dependency_old,records low medium high,Numeric,%,%,5;10;20;30;40;50;60;70,GnBu,
-Median age,Median age,median_age__${sex__slug}__all__${projection__slug},median_age__${sex__slug}__full__${projection__slug},"The median age divides the population into two parts of equal size; that is, there are as many people with ages above the median age as there are with ages below. ${projection__subtitle}",LineChart,,0,true,,,all,none,records low medium high,Numeric,,,15;20;25;30;35;40;45;50;55,GnBu,
+title,Metric Dropdown,ySlugs,tableSlug,subtitle,type,note,yAxisMin,hasMapTab,stackMode,facet,selectedFacetStrategy,_sex,_age_group,_projection,column__type,column__shortUnit,column__unit,column__colorScaleNumericBins,column__colorScaleScheme,column__colorScaleNumericMinValue
```

Comment:
Don't know how you got the `facet` in there, but it's no longer supported (well, it didn't ever do anything, turns out) and we use `selectedFacetStrategy` instead.